.pre-commit-config.yaml

```
@@ -15,59 +15,24 @@ repos:
       - id: requirements-txt-fixer
       - id: trailing-whitespace
 
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.16.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.0
```

```suggestion
    rev: v0.5.1
```

Let's use the latest version.